lib/inspec/profile.rb

```diff
@@ -362,6 +362,22 @@ def check # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metri
         warnings: [],
       }
 
+      # Capture InSpecStyle run
+      runtime_config = Inspec::Config.cached.respond_to?(:final_options) ? Inspec::Config.cached.final_options : {}
+
+      if runtime_config[:inspecstyle]
+        inspecstyle_target = if File.directory?(@target)
+                           File.join(@target, "**/*.rb")
+                         else
+                           @target
+                         end
+
+        cmd = Mixlib::ShellOut.new(
+          "bundle exec rubocop #{inspecstyle_target} -r inspecstyle --only InSpecStyle"
```

Comment:
Can this be changed from a ShellOut to instantiating an instance of InSpecStyle or rubocop within Ruby?
Not shelling out is preferable (though it's more work for us :( )
I believe the use of `bundle exec` means this won't work for people on package installs or running in Habitat.

Reply:
Oooh yeah that would be better. I'll see if I can get our report generated in PureRuby™️